docs/editors/emacs.md

```diff
@@ -129,12 +129,33 @@ To configure Eglot with Metals, replace the `lsp-mode`, `lsp-ui`, `lsp-scala`
 sections from the config above with the settings below.
```

Comment:
Looks like a new version of config snippet has all configuration required for `eglot` instead of original partial snipped so seems like we have to update this description text too

Reply:
new config itself looks good

Reply:
Updated